superset/commands/chart/importers/v1/utils.py

```diff
@@ -55,6 +55,12 @@ def import_chart(
     can_write = ignore_permissions or security_manager.can_access("can_write", "Chart")
     existing = session.query(Slice).filter_by(uuid=config["uuid"]).first()
     if existing:
+        if overwrite and can_write and hasattr(g, "user") and g.user:
```

Comment:
I thought we had a shorthand method for checking whether the user was set, i.e., `get_username()` or similar.

Reply:
we don't exactly, I've created a new one